packages/swingset/src/get-nav-tree.ts

```diff
@@ -46,7 +46,7 @@ export function getNavigationTree(
 
```

Comment:
(nit/non-blocker): For consistency with the method name, what do you think about renaming the file to `get-navigation-tree.ts`?